components/financial_assistance/app/views/financial_assistance/applications/eligibility_results.html.erb

```diff
@@ -54,7 +54,13 @@
             <div class="er-success bg-success">
               <div><i class="fa fa-info-circle fa-lg text-success" aria-hidden="true"></i></div>
               <div class="text-success">
-                <span><b>NEXT STEP:</b> Pick a health insurance plan.</span>
+                <span>
+                  <b>NEXT STEP:</b>
+                  <ul>
+                    <li>If you’re already enrolled in DC Health Link’s Individual & Family plan, <b>we’ve automatically changed your premium. You don’t have to do anything else.</b></li>
```

Comment:
This needs to be `EnrollRegistry[:enroll_app].setting(:short_name).item`, not hard coded to DC Health Link.